core/services/pipeline/graph.go

```diff
@@ -155,11 +151,18 @@ func (n *TaskDAGNode) String() string {
 	return n.dotID
 }
 
+var bracketQuotedAttrRegexp = regexp.MustCompile(`^\s*<([^<>]+)>\s*$`)
```

Comment:
Is a regexp really the right solution here?

Reply:
> This will fail to match something containing brackets within: <{"hi": "hello>world"}>

Yeah, but that's okay, because your example isn't considered valid DOT. The DOT parser will consider the attr string to be terminated at `hello>`, and the following characters will be flagged as a syntax error